.github/workflows/pr-build.yml

```diff
@@ -42,6 +42,18 @@ jobs:
         with: 
           args: build dockerBuildImages
 
+      - name: Determine docker tag
+        id: tag
+        run: echo ::set-output name=tag::$(./gradlew -q printDockerImageDefaultTag | head -1)
+
+      - name: Scan docker image
```

Comment:
It is scanning the container image created just before this step. if we move it to another job then we will have to build docker image again.

Reply:
I guess the tradeoff here is we've already built the image vs the test job is the more appropriate place to run this... test. Given that we want to login in order to build the job though, we'd need a new job anyway so I suppose it's moot. Carry on.

Reply:
The repetition is less concerning to me - we do 90% of that in the test job to run integration tests, for example. The problem is the secret management - whatever job builds the image needs to be in this workflow (specifically, triggered by pull_request_target rather than pull_request which is what test uses), so we'd need a brand new job if we wanted to separate it which isn't worth it.